bin/steps/gdal

```diff
@@ -25,7 +25,7 @@ if (pip-grep -s requirements.txt GDAL gdal pygdal &> /dev/null) then
     echo "-----> Noticed GDAL. Bootstrapping gdal."
     mkdir -p .heroku/vendor
     # Download and extract cryptography into target vendor directory.
-    curl "$VENDORED_GDAL" -s | tar zxv -C .heroku/vendor &> /dev/null
+    curl "$VENDORED_GDAL" --verbose | tar zxv -C .heroku/vendor &> /dev/null
```

Comment:
I like the [pattern](https://github.com/plotly/heroku-buildpack-python/pull/6/commits/08e347eb94e28e718a8625e2506bb6a06df2de37) of showing the verbose output only on failure, I wonder if we could apply it in the `curl`s here in some sort of DRY way. I realise this would take more engineering time, and that it might not be worth it, so I don't feel religiously about this.

Reply:
If this was all our code I'd agree with that, but the code as is doesn't offer a good way to do this (`bin/steps/gdal` and `bin/utils` are separate scripts with no common `source`d libraries). I think repeating the pattern twice will be more maintainable than the alternative.

Reply:
True! Just to be clear I was talking about repeating the pattern of showing the verbose output **only after a failure** but this shouldn't block the merge anyways, so I'll mark this as resolved.